packages/html-reporter/tsconfig.node.json

```diff
@@ -4,5 +4,5 @@
     "module": "esnext",
     "moduleResolution": "node"
   },
-  "include": ["vite.config.ts"]
+  "include": ["vite.config.ts", "bundle.ts"]
```

Comment:
My recollection is that it was making the linter very slow. If it no longer is - fine with me.